api-schema/build.gradle.kts

```diff
@@ -13,3 +13,9 @@ dependencies {
 
   annotationProcessor(libs.lombok)
 }
+
+tasks.test {
+  // Enable parallel test execution
+  systemProperty("junit.jupiter.execution.parallel.enabled", true)
+  systemProperty("junit.jupiter.execution.parallel.mode.default", "concurrent")
```

Comment:
Should we also set `junit.jupiter.execution.parallel.mode.classes.default = concurrent` so the test classes can run in parallel?